### PR TITLE
Noticeboard refactor port + approved stamp

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

[Ports](https://github.com/tgstation/tgstation/pull/59389) noticeboard refactor from TG, also i've made a third button to be able to interact with paper on noticeboard with a pen, in order to write on it, mapped noticeboard in req, as well as added some pens to req area.

Also added green approved stamp, added it in QM's office.

dreamseeker_bzDYkxKWNe

# Explain why it's good for the game

Just compare these two, i think it says it all.

dreamseeker_UW8Us7mk5N
dreamseeker_6O618EfJny

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

dreamseeker_xzHKpzcgsN

dreamseeker_c4OoVNPpNY

dreamseeker_oiz4iP9GNA

</details>

# Changelog
:cl:
add: added approved stamp
fix: fixed being unable to write on papers on noticeboard
ui: changed noticeboard UI from scratch
refactor: refactored noticeboard code
maptweak: added noticeboard to req and approved stamp to QM's office, as well as some pens in req area to interact with noticeboard
/:cl:

### CODE REVIEW

**Comment by fira** [comment]
It still requires pens right? Could you map some in too while at it? As it stands reqs has paper and now the notice board but no pens

**Comment by Blundir** [comment]
> It still requires pens right? Could you map some in too while at it? As it stands reqs has paper and now the notice board but no pens

Added a few

File: maps/map_files/USS_Almayer/USS_Almayer.dmm
Comment (Nanu308) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:1]: lgtm

**Review by Nanu308 — APPROVED** [review]
mapping lgtm

**Review by Nanu308 — DISMISSED** [review]
nvrmind i'm blind, Semiotic symbols, you forgot to add them back

**Comment by Blundir** [comment]
> nvrmind i'm blind, Semiotic symbols, you forgot to add them back

there is simply not much place for them

**Review by fira — APPROVED** [review]

**Pull request merged**